pkg/config/config_test.go

```diff
@@ -96,6 +103,35 @@ var _ = Describe("Config", func() {
 		Expect(cfg.Trunk).To(BeEquivalentTo([]int{5, 19, 20, 21, 22, 23, 55, 101, 102, 103}))
 
 	})
+	It("Assuming correct config file - static bridge config", func() {
+		data := []byte(`{
+		"name": "mynet",
+		"type": "accelerated-bridge",
+		"deviceID": "0000:af:06.1",
+		"bridge": "bridge1, bridge2"
```

Comment:
can add invalid bridge config e.g "bridge": "bridge1," ?
Also a case when the PF attached to invalid bridge but with multi bridge?

Reply:
I added few additional tests-cases